### PR TITLE
EZP-30965: Fixed wrong argument passed to MenuManipulator::moveToPosition() method

### PR DESCRIPTION
**Pull request by hgiesenow**
| Question      | Answer
| ------------- | ---
| Tickets       |  [EZP-30965](https://jira.ez.no/browse/EZP-30965)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Implement tests
- [x] Coding standards (`$ composer fix-cs`)

### CODE REVIEW

File: tests/lib/EventListener/UserMenuListenerTest.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
```
Comment (adamwojs) [inline tests/lib/EventListener/UserMenuListenerTest.php]:
```suggestion
<?php 

/**
 * @copyright Copyright (C) eZ Systems AS. All rights reserved.
 * @license For full copyright and license information view LICENSE file distributed with this source code.
 */
declare(strict_types=1);
```

File: tests/lib/EventListener/UserMenuListenerTest.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+use EzSystems\EzPlatformUser\EventListener\UserMenuListener;
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+/**
+ * @author Hannes Giesenow <hannes.giesenow@elbformat.de>
+ */
+class UserMenuListenerTest extends TestCase
```
Comment (adamwojs) [inline tests/lib/EventListener/UserMenuListenerTest.php]:
```suggestion
final class UserMenuListenerTest extends TestCase
```

File: tests/lib/EventListener/UserMenuListenerTest.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+{
+
+    public function testOnUserMenuConfigureAnonUser()
```
Comment (adamwojs) [inline tests/lib/EventListener/UserMenuListenerTest.php]:
```suggestion
    public function testOnUserMenuConfigureAnonUser(): void
```

File: tests/lib/EventListener/UserMenuListenerTest.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        $factory = $this->getMockBuilder(FactoryInterface::class)->getMock();
+        $menu = $this->getMockBuilder(ItemInterface::class)->getMock();
+        $menu->expects($this->never())->method('reorderChildren');
+        $evt = new ConfigureMenuEvent($factory, $menu);
```
Comment (adamwojs) [inline tests/lib/EventListener/UserMenuListenerTest.php]:
```suggestion
        $event = new ConfigureMenuEvent($factory, $menu);
```

File: tests/lib/EventListener/UserMenuListenerTest.php [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        $tokenStorage = $this->getMockBuilder(TokenStorageInterface::class)->disableOriginalConstructor()->getMock();
+        $uml = new UserMenuListener($tokenStorage);
```
Comment (adamwojs) [inline tests/lib/EventListener/UserMenuListenerTest.php]:
```suggestion
        $listener = new UserMenuListener($tokenStorage);
```

**Comment by hgiesenow** [comment]
Thanks for the review @adamwojs. I rebased my code to your suggestions. Wondering why this was not found by php-cs-fixer :thinking:

**Comment by Nattfarinn** [comment]
Closing in favour of https://github.com/ezsystems/ezplatform-user/pull/65

**Pull request closed without merging**